### ISSUE TITLE
High memory usage?

### ISSUE THREAD

**Issue by stanelie** [Libresonic/libresonic#478]
Hello.

I've noticed that Libresonic is using an unusually large amount of memory. As a comparison, I am running both subsonic 6.0.beta1-KANG and libresonic 6.2 on the same server, having them both share the same media folders.

The memory size of the java process that belongs to subsonic reported by top is 340 MB , and that of libresonic is 920 MB. Is this expected?

How can I tell java to use less memory?

Thanks!

**Comment by hschroedl** [comment]
That's quite funny, just experienced and solved a similar issue. See #465. 
What platform/OS are you using? Are you running Libresonic using Tomcat?

**Comment by protist** [comment]
Yeah, after I resolved #465, I noticed this too. For me, subsonic uses 110 MB and libresonic uses 217 MB.
